infrastructure/lib/variables.ts

```diff
@@ -0,0 +1,8 @@
+export const RepoOwner = 'VikeLabs';
+export const FrontendRepo = 'Passr';
+export const FrontendMainBranch = 'master';
+export const GithubSecretName = 'github';
+export const DomainName = 'passr.ca';
+export const MXSPFRecords = ['"v=spf1 include:spf.improvmx.com ~all"'];
+export const MXReconds = ['10 mx1.improvmx.com', '20 mx2.improvmx.com'];
+export const ProjectPrefix = 'passr';
```

Comment:
It's splitting hairs and comes down to opinion mostly. By always exporting an object we're introducing encapsulation to the module, a public and private interface if you will. We can argue that named exports also provide encapsulation, which they do, but there can be a bit of misdirection when using them in a constants file like this.

To my example of being 250 lines deep into a file and seeing the variable `RepoOwner`, to know what this is and where it comes from, a developer needs to CTRL+F for `RepoOwner`, or use some editor 'find occurences of' functionality or something, then investigate where it comes from. A small change that would make this immediately clearer would be to rename it `REPO_OWNER` which immediately screams "constant", as constants should always be capitalized and snake_case to begin with. But again, where is it coming from. By using default exports in the case of a constants file when a developer sees `domainConstants.REPO_OWNER` 250 lines into a file it's immediately apparent where it's coming from and what it is. This saves the 30 seconds of misdirection.

This is splitting hairs because what is 30 seconds to one or two people, but I suppose on a team of 10 to 100 people, 30 seconds each is a lot of money.

-------------

To the point of exporting 3 different types vs. returning an object that contains 3 different types, I would maybe say don't think about what is in the Object, just consider the fact that when importing any constants file, you _always_ get an Object. This is an honest and consistence interface. This is also splitting hairs because with one or two people it really doesn't matter but when you have a `constants/` directory with 100+ files, the consistency is important.

Months ago at my work we were performing major refactors and we had a random assortment of export types in our constants files, some were named, some weren't, some had both, and the inconsistency was so incredibly rage inducing any time we went to work with constants in different files etc. Having a single API for constants definitely made our code look so much better and made importing constants consistent and clean.

-----------

Again these things aren't really that important right now but I figure I'll point them out because I like to sound like I know what I'm doing 😄